cli-plugins/apps/how-to-guides/how-to-guide-1.hbs.md

```diff
@@ -0,0 +1,28 @@
+# How-to-Guides simple reference
+
+## <a id='create-from-private-repo'></a> Create a workload from a private Git repository
+
+Check how to setup a workload from a Private Git repository in 
+[Out of the box supply chain with private Git repos](../../../scc/building-from-source.hbs.md#private-gitrepository),
+how the supply chain manages these type of workloads in
+[How it works](../../../scc/building-from-source.hbs.md#how-it-works) section and how to
```

Comment:
I think docs require links to be relative to the root instead of relative to the current file
```suggestion
[How it works](scc/building-from-source.hbs.md#how-it-works) section and how to
```